scripts/commerce.js

```diff
@@ -148,52 +201,6 @@ export function getSignInToken() {
   return getCookie('auth_dropin_user_token');
 }
 
-export async function performMonolithGraphQLQuery(query, variables, GET = true, USE_TOKEN = false) {
```

Comment:
Can you really just delete this? I thought there are tools and other things using it.

Reply:
I didn't find anything in the project using it. It was used by the old custom PLP and PDP but that's what fethGraphQl is used for.

Reply:
I found it - this function is sort of a duplicate of the one used in the pdp-metadata tool: https://github.com/hlxsites/aem-boilerplate-commerce/pull/72/files#diff-9ff45fa98b9177c3c5a450134feefedf7db0df3cb9d7b18f7e7ff23ca5b495e2R10

Reply:
If that's the case, that function should be part of the tool since it's the only file using it.

Reply:
> @hannessolo, tagging you since you have built plugins that use Catalog Service. I want to ensure you are not using that function from this file.

No, the commerce picker is completely isolated, it doesn't depend on code from the boilerplate/customer